functions/__fzf_search_git_status.fish

```diff
@@ -1,12 +1,9 @@
 function __fzf_search_git_status --description "Search the git status of the current git repository. Insert the selected file paths into the commandline at the cursor."
-    if not git rev-parse --git-dir >/dev/null 2>&1
+    # Pass configuration color.status=always to force status to use colors even though output is sent to a pipe
+    if not set --local git_color_status (git -c color.status=always status --short 2>/dev/null)
         echo '__fzf_search_git_status: Not in a git repository.' >&2
     else
-        set selected_paths (
-            # Pass configuration color.status=always to force status to use colors even though output is sent to a pipe
-            git -c color.status=always status --short |
-            fzf --ansi --multi
-        )
+        set selected_paths (printf '%s\n' $gitColorStatus | fzf --ansi --multi)
```

Comment:
Still camelCased. Also I think a better variable name would be `git_status_colored`

Reply:
Ah my bad, wasn't thorough enough sorry. Will do!